ios/Plugin/Email/Email.swift

```diff
@@ -33,16 +33,27 @@ public class Email {
     ///   - account: An instance of the Account class containing user and account information.
     ///   - onError: A closure to handle error messages.
     ///   - onSuccess: A closure to handle success actions.
-    public func login(_ account: Account, onError: @escaping (String) -> Void, onSuccess: @escaping () -> Void)  {
+    public func login(_ account: Account, onError: @escaping (String, RspErrorEnum) -> Void, onSuccess: @escaping () -> Void)  {
         let email = BRIMAPAccount(provider: .gmailIMAP, email: account.user, password: account.password!)
         DispatchQueue.main.async {
             let rootVc = UIApplication.shared.windows.first?.rootViewController
             BREReceiptManager.shared().setupIMAP(for: email, viewController: rootVc!, withCompletion: { result in
                 BREReceiptManager.shared().verifyImapAccount(email, withCompletion: { success, error in
-                    if success {
-                        onSuccess()
-                    } else {
-                        onError(error.debugDescription)
+                    guard let errorCallback: Int = (error as? NSError)?.code else {
```

Comment:
the name of this variable is not clear. Looks like it is a callback, but it is just the error returned.

Reply:
Resolved